### PR TITLE
Update secret-get to use the new syntax

### PR DESCRIPTION
**Pull request by wallyworld**
The secret-get hook command is updated to use the new CLI syntax.
`--update`, `--peek`, and `--label` options are added.
The secret value implementation is also adjusted to suit.

A followup PR will wire up the backend.

## Checklist

- [x] Code style: imports ordered, good names, simple structure, etc
- [x] Comments saying why design decisions were made
- [x] Go unit tests, with comments saying what you're testing
- ~[ ] [Integration tests](https://github.com/juju/juju/tree/develop/tests), with comments saying what you're testing~
- [x] [doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) added or updated in changed packages

## QA steps

Just unit tests for this PR.

### CODE REVIEW

File: worker/uniter/runner/jujuc/secret-get_test.go [hunk 1]
```diff
@@ -22,58 +22,99 @@ type SecretGetSuite struct {
...
-	c.Assert(bufferString(ctx.Stdout), gc.Equals, `{"key":"s3cret!"}`+"\n")
+	c.Assert(bufferString(ctx.Stdout), gc.Equals, `
+cert: cert
+key: key
+`[1:])
 }
 
-func (s *SecretGetSuite) TestSecretGetSingularEncoded(c *gc.C) {
+func (s *SecretGetSuite) TestSecretGetBinary(c *gc.C) {
+	encodedValue := `R0lGODlhDAAMAIQAAP//9/X17unp5WZmZgAAAOfn515eXvPz7Y6OjuDg4J+fn5OTk6enp56enmlpaWNjY6Ojo4SEhP/++f/++f/++f/++f/++f/++f/++f/++f/++f/++f/++f/++f/++f/++SH+Dk1hZGUgd2l0aCBHSU1QACwAAAAADAAMAAAFLCAgjoEwnuNAFOhpEMTRiggcz4BNJHrv/zCFcLiwMWYNG84BwwEeECcgggoBADs=`
```
Comment (tlm) [inline worker/uniter/runner/jujuc/secret-get_test.go:94]: Is this valid base64?

**Review by tlm — APPROVED** [review]

File: worker/uniter/runner/jujuc/secret-get_test.go (hunk 1 above)
Comment (wallyworld) [reply worker/uniter/runner/jujuc/secret-get_test.go:94]: yes, it's an encoded GIF icon

File: api/agent/secretsmanager/client_test.go [hunk 2]
```diff
@@ -197,7 +200,7 @@ func (s *SecretsSuite) TestGetSecretsError(c *gc.C) {
 		return nil
 	})
 	client := secretsmanager.NewClient(apiCaller)
-	result, err := client.GetValue("secret:9m4e2mr0ui3e8a215n4g")
+	result, err := client.GetValue("secret:9m4e2mr0ui3e8a215n4g", "", true, true)
 	c.Assert(err, gc.ErrorMatches, "boom")
```
[the remark this answers is not in the document]
Comment (wallyworld) [reply api/agent/secretsmanager/client_test.go:204]: it's just a string in the test
```
Error: &params.Error{Message: "boom"},
```

**Pull request merged**